docs/installation/importing-css-assets-and-javascript.md

```diff
@@ -0,0 +1,162 @@
+# Import CSS, assets and JavaScript
+
```

Comment:
Should we say that this page is more relevant for production sites and for ensuring that your assets stay up to date? It feels that the relationship of this page to the "Get started" isn't really clear. A user might feel a bit overwhelmed if they hit this first 😅 

We could also include a link to the "Get started" guidance? 

Possibly both of these things will be taken care of by the navigation in tech docs template.

Reply:
This page is trying to do a lot, it's true! It's also aiming to be something of a 'fix issues with CSS, assets and JavaScript' page. In a future iteration we need to tease the content apart a little (possibly into separate pages).

I think you're right that the navigation should hopefully help here. Get started will be nearer the top (and immediately under install) in the nav, so I hope users will be drawn back to Get started if this looks overwhelming. 

So would we be ok to leave as is for now, but aim to test a little in Q1?

Reply:
Yes sounds good, thanks @m-green 👍